app/mainwindow2.h

```diff
@@ -91,6 +91,8 @@ public slots:
 
     void importMovie();
 
+    void lockWidgets( bool shouldLock );
```

Comment:
Actually I just found out it’s *not* only syntactic sugar, it affects the generation of the meta object. That means that the way it currently is only `&MainWindow2::lockWidgets` will work in `connect()` calls, but `SIGNAL(lockWidgets(bool))` will not work because `SIGNAL()` relies on the meta object which currently doesn’t know that lockWidgets is a slot.